modules/reference/pages/rpk/rpk-debug/rpk-debug-bundle.adoc

```diff
@@ -104,25 +99,22 @@ supported, e.g. 3MB, 1GiB (default "20MB").
 
 |-l, --label-selector |stringArray |Comma-separated label selectors to filter your resources. e.g: <label>=<value>,<label>=<value> (k8s only) (default [app.kubernetes.io/name=redpanda]).
 
-|--logs-since |string |Include log entries on or newer than the
-specified date in journalctl date format, for example YYYY-MM-DD.
+|--logs-since |string |Include logs dated from specified date onward. (`journalctl` date format: `YYYY-MM-DD`, `yesterday`, or `today`). Refer to xref:https://man7.org/linux/man-pages/man1/journalctl.1.html[journalctl documentation] for more options (default `yesterday`).
```

Comment:
```suggestion
|--logs-since |string |Include logs dated from specified date onward. This flag accepts a `journalctl` date format such as `YYYY-MM-DD`, `yesterday`, or `today`. Refer to xref:https://man7.org/linux/man-pages/man1/journalctl.1.html[`journalctl` documentation] for more options (default `yesterday`).
```